gef-ui/templates/partials/amendments/facility-end-date.njk

```diff
@@ -0,0 +1,115 @@
+{% extends "index.njk" %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% import '../../_macros/what-is-a-facility-end-date.njk' as whatIsAFacilityEndDate %}
+
+{% block pageTitle -%}
+  Facility end date
+{%- endblock %}
+
+{% set headingHtml %}
+<header>
+  <span class="govuk-caption-l" data-cy="heading-caption">
+    {{ exporterName }}
+  </span>
+
+  <h1 class="govuk-label-wrapper" data-cy="page-heading">
+    <label class="govuk-label govuk-label--xl govuk-!-padding-bottom-3" for="facility-end-date">
+      Facility end date
+    </label>
+  </h1>
+</header>
+{% endset %}
+
+{% block content %}
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      },
+      classes: "govuk-!-margin-top-4 govuk-!-margin-bottom-4"
+    }) }}
+  {% endif %}
+
+  {{ govukBackLink({
```

Comment:
Would it be prudent to have a back link commonised template to make the code DRY?